File: Server/Server/Models/Inside/Question.cs
namespace Server.Models.Inside
{
    public class Question
    {
        public int Id { get; set; }
        public string Title { get; set; } = string.Empty;
        public int QuestionTypeId { get; set; }
        public int QuizId { get; set; }
        public virtual QuestionType QuestionType { get; set; } = null!;
        public virtual ICollection<Answer> Answers { get; set; } = null!;
        public virtual Quiz Quiz { get; set; } = null!;

    }
}

File: Client/src/app/models/question.ts
import { Answer } from "./answer";
import { QuestionType } from "./questionType";
import { Quiz } from "./quiz";

export class Question {

    id: number = -1;
    title: string = "";
    isCorrect: boolean = false;
    questionTypeId: number = -1;
    quizId: number = -1;
    answers: Array<Answer> = [];
    questionType!:QuestionType;
    quiz!:Quiz;
}